README.md

```diff
@@ -110,7 +110,7 @@ published here.  This has several implications for the foreseeable future:
 ## How to Report Issues
 
 Problems specific to the Open GPU Kernel Modules can be reported in the
-Issues section of the https://github.com/NVIDIA/open-gpu-kernel-modules
+'Issues Section' of the https://github.com/NVIDIA/open-gpu-kernel-modules
```

Comment:
```suggestion
'Issues' section of the https://github.com/NVIDIA/open-gpu-kernel-modules
```
The section is called “Issues”, thus it makes no sense to capitalize “section” and put it in quotation marks.

“Issues Section” would imply that the place to put the issues is actually called Issues Section like this:
![image](https://user-images.githubusercontent.com/32276754/168053071-8cd2a28c-0160-451b-a013-aa730fc9887b.png)